### ISSUE TITLE
V2 dataset images

### ISSUE THREAD

**Issue by mayankbhagya** [openimages/dataset#36]
Are those the same as v1 images?
Or has the whole set of images also changed?

**Comment by rkrasin** [comment]
@mayankbhagya same images; more image-level labels; bounding boxes are added.
One important (and a bit unfortunate) change: v2 validation + test set = v1 validation set

@nalldrin please, correct me, if I am wrong.

**Comment by mayankbhagya** [comment]
I think you're right @rkrasin 
Just came across the [official announcement post](https://research.googleblog.com/2017/07/an-update-to-open-images-now-with.html).
From the post:

> Today, we introduce an update to Open Images, which contains the addition of a total of ~2M bounding-boxes to the **existing dataset**

**Comment by nalldrin** [comment]
Indeed we split the validation set into validation and test. The reason is that 167k images is too large to easily compute metrics during training.

Re: images, the set of images is identical to the v1 release.
